tools/pyqbdi/binding/Memory.cpp

```diff
@@ -0,0 +1,121 @@
+/*
+ * This file is part of pyQBDI (python binding for QBDI).
+ *
+ * Copyright 2017 Quarkslab
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "pyqbdi.hpp"
+
+namespace QBDI {
+namespace pyQBDI {
+
+void init_binding_Memory(py::module& m) {
+
+    py::enum_<QBDI::Permission>(m, "Permission", py::arithmetic(), "Memory access rights.")
+        .value("PF_NONE", QBDI::Permission::PF_NONE, "No access")
+        .value("PF_READ", QBDI::Permission::PF_READ, "Read access")
+        .value("PF_WRITE", QBDI::Permission::PF_WRITE, "Write access")
+        .value("PF_EXEC", QBDI::Permission::PF_EXEC, "Execution access")
+        .export_values()
```

Comment:
Move a the end

Reply:
`py::enum::def` return a `py::class`. The method `export_values` cannot be call on a `py::class`. We must call `export_values` before define `__str__` method.